runners/jstorm/pom.xml

```diff
@@ -0,0 +1,201 @@
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  
+  <artifactId>beam-runners-jstorm</artifactId>
+
+  <name>Apache Beam :: Runners :: JStorm</name>
+
+  <packaging>jar</packaging>
+
+  <properties>
+    <jstorm.core.version>2.2.1</jstorm.core.version>
+  </properties>
+  
+  <profiles>
+    <profile>
+      <!-- This profile adds execution of ValidatesRunner integration tests
+           against a JStorm local cluster. -->
+      <id>local-validates-runner-tests</id>
+      <activation><activeByDefault>false</activeByDefault></activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>validates-runner-tests</id>
+                <phase>integration-test</phase>
+                <goals>
+                  <goal>test</goal>
+                </goals>
+                <configuration>
+                  <groups>
+                    org.apache.beam.sdk.testing.ValidatesRunner
+                  </groups>
+                  <excludedGroups>
+                    org.apache.beam.sdk.testing.UsesStatefulParDo,
+                    org.apache.beam.sdk.testing.UsesTimersInParDo,
+                    org.apache.beam.sdk.testing.UsesSplittableParDo,
+                    org.apache.beam.sdk.testing.UsesCommittedMetrics
+                  </excludedGroups>
+                  <parallel>none</parallel>
+                  <forkCount>1</forkCount>
+                  <reuseForks>false</reuseForks>
+                  <failIfNoTests>true</failIfNoTests>
+                  <dependenciesToScan>
+                    <dependency>org.apache.beam:beam-sdks-java-core</dependency>
+                  </dependenciesToScan>
+                  <systemPropertyVariables>
+                    <beamTestPipelineOptions>
+                      [
+                        "--runner=org.apache.beam.runners.jstorm.JStormRunner"
+                      ]
+                    </beamTestPipelineOptions>
+                    <beam.spark.test.reuseSparkContext>true</beam.spark.test.reuseSparkContext>
+                    <spark.ui.enabled>false</spark.ui.enabled>
+                    <spark.ui.showConsoleProgress>false</spark.ui.showConsoleProgress>
+                  </systemPropertyVariables>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+  </profiles>
+
+  <dependencies>
+    <!-- JStorm dependencies -->
+    <dependency>
+      <groupId>com.alibaba.jstorm</groupId>
+      <artifactId>jstorm-core</artifactId>
+      <version>${jstorm.core.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    
+    <!-- Beam dependencies -->
+    <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-sdks-java-core</artifactId>
+      <exclusions>
```

Comment:
Comment on each exclusion, even if duplicated it makes it clear to someone reviewing just a few lines.